### ISSUE TITLE
Problems when trying to finetune with one class

### ISSUE THREAD

**Issue by mew124** [wizyoung/YOLOv3_TensorFlow#202]
Hi, 

I'm pretty new to machine learning and need some help.
I am trying to finetune on a dataset with only one labeled class. Can I still use the pretrained weights? I plotted the losses in tensorboard. If I don't change anything and finetune with my labeled data, the total loss goes up. I guess the reason might be that it detects other classes as well that are not labeled in my dataset... I tried to change the number of classes in the args file, but I get  an error that the the model does not fit to the weights. 

Can someone please give me some advice how to finetune correctly if only one of the 80 COCO classes is labeled in my datset?
Many thanks in advance!!

**Comment by FuNian788** [comment]
If your dataset's class is in 'coco.names', eg 'car' which is the 3rd class, you may change Line 128 in './eval.py' to:
> _for  ii in [2]:_
